spring/pom.xml

```diff
@@ -0,0 +1,170 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.jboss.narayana.quickstarts.spring</groupId>
+    <artifactId>spring-all</artifactId>
+    <version>5.2.3.Final-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>Narayana Quickstarts: JBTM and Spring</name>
+    <description>Examples show how to integrate with Spring</description>
+
+    <url>http://jboss.org/jbosstm</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <version.maven-compiler-plugin>3.1</version.maven-compiler-plugin>
+        <version.maven-surefire-plugin>2.9</version.maven-surefire-plugin>
+        <version.org.springframwork>4.2.0.RELEASE</version.org.springframwork>
+        <version.org.jboss.narayana>${project.version}</version.org.jboss.narayana>
+        <version.org.jboss.logging.jboss-logging>3.1.2.GA</version.org.jboss.logging.jboss-logging>
+        <version.log4j>1.2.16</version.log4j>
+        <version.org.jboss.spec.javax.transaction>1.0.1.Final</version.org.jboss.spec.javax.transaction>
+        <version.org.jboss.jboss-transaction-spi>7.1.1.Final</version.org.jboss.jboss-transaction-spi>
+        <version.com.h2database>1.3.171</version.com.h2database>
+        <version.junit>4.10</version.junit>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Maven Repository Group</name>
+            <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <!-- Spring dependencies -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>${version.org.springframwork}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-tx</artifactId>
+            <version>${version.org.springframwork}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-jdbc</artifactId>
+            <version>${version.org.springframwork}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>${version.org.springframwork}</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- Narayana dependencies -->
+        <dependency>
+            <groupId>org.jboss.narayana.jta</groupId>
+            <artifactId>narayana-jta</artifactId>
```

Comment:
Good observation @tomjenkinson - this also caught me out whilst doing the benchmark comparisons.